src/gui/main.cpp

```diff
@@ -417,24 +412,34 @@ int main(int argc, char **argv)
         }
     });
 
+    platform->startServices();
+
+    // Handle user requests from the command-line first, before checking for updates. Because, if
+    // the user explicitly requested an action, then quiting because of an update will not be
+    // appreciated.
     if (options.show) {
         ocApp->gui()->slotShowSettings();
-    }
-    if (!options.fileToOpen.isEmpty()) {
+        // The user explicitly requested the settings dialog, so don't start the new-account wizard.
+    } else if (!options.fileToOpen.isEmpty() && !AccountManager::instance()->accounts().isEmpty()) {
+        // Only try to open a file when accounts have been configured.
         QTimer::singleShot(0, ocApp.get(), [ocApp = ocApp.get(), fileToOpen = options.fileToOpen] { ocApp->openVirtualFile(fileToOpen); });
+    } else {
+        // No user-requested action, check for updates.
+#ifdef WITH_AUTO_UPDATER
+        // if handleStartup returns true, main()
+        // needs to terminate here, e.g. because
+        // the updater is triggered
+        Updater *updater = Updater::instance();
+        if (updater && updater->handleStartup()) {
+            return 1;
+        }
+#endif
     }
 
-    platform->startServices();
-
-#ifdef WITH_AUTO_UPDATER
-    // if handleStartup returns true, main()
-    // needs to terminate here, e.g. because
-    // the updater is triggered
-    Updater *updater = Updater::instance();
-    if (updater && updater->handleStartup()) {
-        return 1;
+    // Display the wizard if we don't have an account yet, and no other UI is showing.
+    if (!options.show && AccountManager::instance()->accounts().isEmpty()) {
```

Comment:
why: `!options.show` ?
This appears to break the gui tests.
options.show will only bring the application to the front, which will be done by the wizard anyhow.

Reply:
Ok, removed that part. Let's see how the tests do now.